nav2_map_server/package.xml

```diff
@@ -25,6 +24,10 @@
   <depend>nav2_msgs</depend>
   <depend>nav2_util</depend>
   <depend>graphicsmagick</depend>
+  <depend>sensor_msgs</depend>
+  <depend>geometry_msgs</depend>
+  <depend>eigen</depend>
+  <depend>pcl_conversions</depend>
```

Comment:
PCL isn't on this list, do you use PCL directly? If not, remove from the CMakeLists includes if you just needed pcl_conversions